lang/cs/Org.Apache.REEF.Client/YARN/YarnCommandLineEnvironment.cs

```diff
@@ -87,7 +96,7 @@ internal string GetYarnCommandPath()
         /// </summary>
         /// <param name="arguments"></param>
         /// <returns>Whatever was printed to stdout by YARN.</returns>
-        internal string Yarn(params string[] arguments)
+        private string Yarn(params string[] arguments)
```

Comment:
This is tricky, as this code would have to run on the client side, right? And don't we want to rid ourselves from the Java dependency there?

Or, put it another way: If we specialize to HDI, this should not be needed, right?